src/reference/sectionB_elements.inc

```diff
@@ -56,11 +56,11 @@ An :code:`import` element information item (referred to in this specification as
 
    1. Every :code:`import` element MUST contain an attribute in the namespace :code:`http://www.w3.org/1999/xlink`, with a local name equal to :code:`href`.
 
-      The value of this attribute SHALL be a valid locator :code:`href`, as defined in :href_locator:`Section 5.4 of the XLink specification <>` [:ref:`6<ref_xlink>`].
+      The value of this attribute SHALL be a valid locator :code:`href`, as defined in :xlink:`Section 5.4 <#link-locators>` of the :xlink:`XLink specification <>` [:ref:`6<ref_xlink>`].
 
-      The :code:`href` attribute SHALL be treated according to the :xlink:`XLink specification` [:ref:`6<ref_xlink>`], by applying the rules for simple-type elements.
+      The :code:`href` attribute SHALL be treated according to the :xlink:`XLink specification <>` [:ref:`6<ref_xlink>`], by applying the rules for simple-type elements.
 
-      When describing an :code:`import` element or one of its children, the phrase "imported CellML infoset" SHALL refer to the :ref:`CellML infoset<specA_cellml_infoset>` obtained by parsing the document referenced by the :code:`href` attribute.
+      When describing an :code:`import` element or one of its children, the phrase "imported CellML infoset" SHALL refer to the :ref:`CellML infoset<specA_cellml_infoset>` obtained by parsing the document referenced by the :code:`href` attribute value.
```

Comment:
Given that we argued over having these "value" things in over and over again, I don't think we should be making these changes at the last minute, without discussion?

Reply:
Eek, did we?? I was putting them in because I thought I was following what you started :/ ... oops ... they're all in one commit if that was the wrong thing to do.

Reply:
5853b407c41c2f41f1dfa03dd1726a8e99475e9e

Reply:
So, shall I revert that commit then?

Reply:
Damn, conflicts now... Argh!